resources/views.py

```diff
@@ -0,0 +1,14 @@
+from django.shortcuts import render
+from .models import Resource
+
+
+def resources(request):
+    """ Display the useful resources and links page. """
+
+    template = "resources/resources.html"
+    resources = Resource.objects.all()
+    context = {
```

Comment:
Not sure you need to create an extra `context` dict here just for one key. I would just pass an `{'resources': resources}` into the render function directly.